canvas.h

```diff
@@ -235,7 +235,13 @@ class Canvas : public QWidget
 	/**
 	 * @brief Checks all the tracks in the map class to see if they are all connected with no unconnected ends.
 	 */
-	void checkAllTracksInMapConnected();
+	bool checkAllTracksInMapConnected();
+
+	/**
+	 * @brief Get total number of tracks.
+	 * @return tracks Number of tracks as an int.
+	 */
+	int getTrackTotal();
```

Comment:
Recommend `int getTrackTotal() const;`

Reply:
(I see you did this in other places)

Reply:
Yep probs laziness on my part, will fix.